src/app/core/config-setup/config-import-parser.service.ts

```diff
@@ -0,0 +1,144 @@
+import { Injectable } from "@angular/core";
+import { EntityConfig } from "../entity/entity-config.service";
+import { EntityListConfig } from "../entity-components/entity-list/EntityListConfig";
+import { EntityDetailsConfig } from "../entity-components/entity-details/EntityDetailsConfig";
+import {
+  CONFIGURABLE_ENUM_CONFIG_PREFIX,
+  ConfigurableEnumConfig,
+} from "../configurable-enum/configurable-enum.interface";
+import { EntitySchemaField } from "../entity/schema/entity-schema-field";
+import { ConfigService } from "../config/config.service";
+import { ConfigFieldRaw } from "./config-field.raw";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ConfigImportParserService {
+  enumsAvailable: Map<string, ConfigurableEnumConfig> = new Map();
+  existingEnumHashmap: Map<string, string> = new Map();
+
+  constructor(private configService: ConfigService) {
+    const enums = this.configService.getAllConfigs(
+      CONFIGURABLE_ENUM_CONFIG_PREFIX
+    );
+    // TODO: how to get the id?
```

Comment:
```suggestion
    enums.forEach((enum) => this.enumsAvailable.set(enum._id, enum));
```

Reply:
but enum configs don't have _id, do they? they are just an array of options

Reply:
The function `getAllConfigs` automatically sets this `_id` property.

Reply:
Ok probably I am wrong, the enums are arrays and I actually have no idea what happens with the assignment of the `_id` property of that array. At least it doesn't throw an error 😅